### PR TITLE
Added two suggestion components!

### PR DESCRIPTION
**Pull request by Joseph765**
I've created a new function that gets 6 random items from my database. 
PR1

This function looks complicated for 3 reasons: 

1). I wanted to make sure that the random image urls from my database never repeated themselves, that's why there's a usedImages array and calling the function recursively when that url has already been used.
2). I have to do two queries if I want to get the shop name as well. 
3). I attach each result from the different queries to an array so that I can have one array with all the info my front end needs. Also I attach the shop name to the previous object result (the shop name is in a seperate table). The end result looks something like this except there's 6 items: 

`[
  { 
   id: 1, name: "Face Mask", shipping: "Free shipping", 
   price: "$352.34", shop_id: 1, shop_name: "Masks R Us" 
  }
 ]`

PR2

Above is my component I made to display the 6 random items their respective shop names as well (I cropped parts out of my component so look at the actual code to see the rest). I actually reused this component twice for the "You may also like" and "Popular right now" sections:

PR3

### CODE REVIEW

**Comment by Malyarc** [comment]
if it displays the data correctly then LGTM!

**Review by mta630 — APPROVED** [review]
lgtm, nice pull request!

File: dist/app.bundle.js [hunk 1]
```diff
@@ -223,34 +223,9 @@ eval("\n\nif (false) {} else {\n  module.exports = __webpack_require__(/*! ./cjs
...
-
-/***/ }),
+/***/ (function(module, exports) {
```
Comment (therobinkim) [inline dist/app.bundle.js:226]: never commit your bundle

Comment (therobinkim) [inline dist/app.bundle.js:226]: never commit your bundle

File: report.20200506.165027.668.0.001.json [hunk 2]
```diff
@@ -0,0 +1,452 @@
+
+{
```
Comment (therobinkim) [inline report.20200506.165027.668.0.001.json:2]: I'm not sure why this file was committed

File: server/api.js [hunk 3]
```diff
@@ -19,6 +19,48 @@ const apiMethods = {
       }
     });
   },
+  get6Random: (callback) => {
+    const results = [];
+    let counter = 0;
+    const usedImages = [];
+
+    function getRandomProducts(index) {
+      db.query(`SELECT * FROM products WHERE id = ${Math.floor(Math.random() * 100) + 1}`, (err, result) => {
```
Comment (therobinkim) [inline server/api.js:28]: I don't think there should be any randomness involved w/ info retrieval

File: server/api.js [hunk 4]
```diff
@@ -19,6 +19,48 @@ const apiMethods = {
...
+              lastResult.shop_name = JSON.parse(JSON.stringify(shopResult))[0].name;
+              let shopNameCounter = 0;
+              for (let i = 0; i < results.length; i += 1) {
+                if (results[i].shop_name) {
+                  shopNameCounter += 1;
+                }
+              }
+              if (shopNameCounter === 6 && counter === 0) {
+                counter += 1;
+                callback(null, results);
```
Comment (therobinkim) [inline server/api.js:52]: looks like an opportunity to use promises

File: src/components/Suggested.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    });
+  }
+
+  useEffect(() => {
+    if (counter === 1) {
+      counter += 1;
+      get();
+    }
+  }, [counter]);
```
Comment (therobinkim) [inline src/components/Suggested.jsx:31]: is the purpose of this code to only work w/ data from the 2nd get request? if so, what purpose does that serve?

File: src/components/Suggested.jsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  const suggestedNodes = suggestionItems.map( item => {
+    return (
+      <div className="suggestedItm">
+        <img src={item.image_url} />
+        <p className="itemName" >{item.name}</p>
+        <div className="shopName-small">{item.shop_name}</div>
+        <div className="price">{item.price}</div>
+        <div className={item.shipping.split(' ').join('')} >{item.shipping}</div>
```
Comment (therobinkim) [inline src/components/Suggested.jsx:40]: pretty neat way to generate class names! do you have any examples of what potential class names might end up being?

**Pull request merged**